routing/cross_mwm_connector_serialization.hpp

```diff
@@ -369,7 +376,17 @@ class CrossMwmConnectorSerializer final
     std::vector<Section> m_sections;
   };
 
-  static uint32_t CalcBitsPerOsmId(std::vector<Transition> const & transitions);
+  // @todo(bykoianko) This method should be implement in another way. Having template param
+  // |CrossMwmId| it should not called CalcBitsPerOsmId().
+  template <typename CrossMwmId>
+  static uint32_t CalcBitsPerOsmId(std::vector<Transition<CrossMwmId>> const & transitions)
```

Comment:
It surely should not be called CalcBitsPerOsmId. Could you please rename it to CalcBitsPerCrossMwmId?

Reply:
Done